src/librustc_span/source_map.rs

```diff
@@ -710,7 +710,7 @@ impl SourceMap {
     pub fn next_point(&self, sp: Span) -> Span {
         let start_of_next_point = sp.hi().0;
 
-        let width = self.find_width_of_character_at_span(sp, true);
```

Comment:
This is the only necessary change to stop the ICEs in stable and beta.

Reply:
This isolated change is stable-accepted